keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -695,102 +697,74 @@ traffic between tenant Namespaces and `kube-system`. For `coredns` etc. to work,
 `kube-system` Namespace or at least the `app=kube-dns` pods needs to be added into
 the `Allow` rule list.
 
-#### Story 3: Isolate multiple tenants in a cluster
+#### Sample Spec for Story 3: Isolate multiple tenants in a cluster
 
 As a cluster admin, I want to isolate all the tenants (modeled as Namespaces)
 on my cluster from each other by default.
 
 ```yaml
-apiVersion: networking.k8s.io/v1alpha1
-kind: ClusterDefaultNetworkPolicy
+apiVersion: netpol.networking.k8s.io/v1alpha1
+kind: ClusterNetworkPolicy
 metadata:
-  name: namespace-isolation
+  name: namespace-isolation # strictly deny inter-namespace traffic for tenant namespaces with the exception
+                            # of system namespaces and intra-namespace traffic. Tenants are empowered with
+                            # the control of intra-namespace traffic and from system namespaces like kube-system.
 spec:
   appliedTo:
     namespaceSelector:
       matchLabels:
-        type: tenant  # assuming all tenant namespaces will be created with this label
+        type: tenant      # assuming all tenant namespaces will be created with this label
   ingress:
-    - onlyFrom:
+    - from:
       - namespaces:
-          self: true
+          scope: Self
+        action: Empower   # add exception to deny rule for intra-namespace traffic
+      - namespaceSelector:
+          matchLabels:
+            app: system
+        action: Empower   # add exception to deny rule for system namespaces like kube-system
+      - action: Deny      # deny traffic from all namespaces except for exceptions provided by Empower rules
+        namespaceSelector: {}
 ```
 
-__Note:__ The above policy will take no effect if applied together with
-`ingress-egress-gateway`, since both policies apply to the same Namespaces, and
-ClusterNetworkPolicy rules have higher precedence than ClusterDefaultNetworkPolicy
-rules.
-In this policy, tenant isolation (Namespace being the boundary) is not enforced,
-but rather provided as a cluster default for initial security postures in each
-tenant. Tenants can completely overwrite this based on the need of its namespaces,
-using NetworkPolicy. They cannot, however, overwrite Allow, Deny and Empower
+In this policy, tenant isolation (Namespace being the boundary) is strictly enforced.
+Tenants can however allow or deny intra-namespace traffic and traffic from "kube-system"
+Namespace depending on their needs. They cannot, however, overwrite Allow and Deny
 rules which cluster admins listed out as guardrails (dns must be allowed, egress
-to some IPs must be denied etc.) If strict tenant isolation is desired, cluster
-admins should write a ClusterNetworkPolicy instead. See Story 4 below.
+to some IPs must be denied etc.)
 
-#### Story 4: Enforce network/security best practices
+#### Sample Spec for Story 4: Enforce network/security best practices
```

Comment:
I wonder if we should revise the name of this story now. Instead of `Enforce network/security best practices`, it feels more like something in the lines of `Set zero-trust default security posture for tenants`

Reply:
done